gatsby/src/components/measure-list/measure.module.scss

```diff
@@ -33,7 +33,10 @@
   border-color: #2362a2;
   color: #2362a2;
 
-  &:hover {
-    background-color: #2362a2;
+  &:hover,
+  &:focus {
+    border-color: #254e80;
```

Comment:
maybe we could use variables for brand colors?

Reply:
those variables are inside our design system (base/_variables)

Reply:
@vitkolos I am just fix hover effect, can you create new ticket for use variables in module styles?

Reply:
it's there

Reply:
https://trello.com/c/n8t31meC/79-setup-ds-variables-in-module-styles